### PR TITLE
clean up dependencies

### PR DESCRIPTION
**Pull request by DeNeutoy**
None of these dependencies are used in the library anymore.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -48,7 +48,6 @@ gevent>=1.3.6
 
 # For text normalization
 ftfy
-word2number>=1.1
```
Comment (matt-gardner) [inline requirements.txt]: The RC code hasn't been removed yet, has it?

Comment (matt-gardner) [reply requirements.txt]: These are all done in #3348.

Comment (DeNeutoy) [reply requirements.txt]: Oops, sorry.

Comment (DeNeutoy) [reply requirements.txt]: Fair play, I noticed because the SQL ones got missed from the semparse stuff. i'll close this

**Pull request closed without merging**